utils/progressbar/progressbar.go

```diff
@@ -137,71 +143,16 @@ func createSpinnerFramesArray() []string {
 	return spinnerFramesArray
```

Comment:
The white color in this function should be changed to green to match the change in the reader progress bar